src/lib.rs

```diff
@@ -248,56 +251,15 @@ where
     }
 }
 
-/// An element locator.
-///
-/// See [the specification](https://www.w3.org/TR/webdriver1/#locator-strategies) for more details.
-#[derive(Clone, Copy, Eq, PartialEq, Ord, PartialOrd, Debug, Hash)]
-pub enum Locator<'a> {
-    /// Find an element matching the given [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors).
-    Css(&'a str),
-
-    /// Find an element using the given [`id`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id).
-    Id(&'a str),
-
-    /// Find a link element with the given link text.
-    ///
-    /// The text matching is exact.
-    LinkText(&'a str),
-
-    /// Find an element using the given [XPath expression](https://developer.mozilla.org/en-US/docs/Web/XPath).
-    ///
-    /// You can address pretty much any element this way, if you're willing to put in the time to
-    /// find the right XPath.
-    XPath(&'a str),
-}
-
-impl<'a> From<Locator<'a>> for webdriver::command::LocatorParameters {
-    fn from(locator: Locator<'a>) -> webdriver::command::LocatorParameters {
-        match locator {
-            Locator::Css(s) => webdriver::command::LocatorParameters {
-                using: webdriver::common::LocatorStrategy::CSSSelector,
-                value: s.to_string(),
-            },
-            Locator::Id(s) => webdriver::command::LocatorParameters {
-                using: webdriver::common::LocatorStrategy::XPath,
-                value: format!("//*[@id=\"{}\"]", s),
-            },
-            Locator::XPath(s) => webdriver::command::LocatorParameters {
-                using: webdriver::common::LocatorStrategy::XPath,
-                value: s.to_string(),
-            },
-            Locator::LinkText(s) => webdriver::command::LocatorParameters {
-                using: webdriver::common::LocatorStrategy::LinkText,
-                value: s.to_string(),
-            },
-        }
-    }
-}
-
-mod client;
+pub mod client;
```

Comment:
Why make this module public?

Reply:
This was to make new `pub` types inside this module reachable, without necessity to polute root namespace with them (as they're quite specific).